spec/koans.js

```diff
@@ -380,23 +390,23 @@ describe('arrow functions. ', () => {
       let bound = new LexicallyBound();
       let fn = bound.getFunction();
 
-      //expect(fn()).toBe(bound);
+      expect(fn()).toBe(bound);
     });
 
     it('can NOT bind a different context', function() {
       let bound = new LexicallyBound();
       let fn = bound.getFunction();
       let anotherObj = {};
-      let expected = anotherObj; //change this
+      let expected = bound;
 
-      //expect(fn.call(anotherObj)).toBe(expected);
+      expect(fn.call(anotherObj)).toBe(expected);
     });
 
     it('`arguments` doesnt work inside arrow functions', function() {
       let bound = new LexicallyBound();
       let fn = bound.getArgumentsFunction();
 
-      //expect(fn(1, 2).length).toEqual(0);
+      expect(fn(1, 2).length).toEqual(2);
```

Comment:
Yes, but the point here is to show that the 'arguments' keyword does not work inside arrow functions, so if you'd write your getArgumentsFunction() like that : 
```js
    getArgumentsFunction() {
      return () => {
        return arguments;
      };
    }
```
you would still have 
```js
 expect(fn(1, 2).length).toEqual(0);
```